src/test/resources/History.feature

```diff
@@ -0,0 +1,17 @@
+Feature: View History
+  As a user I want to see List of Appointments I have created
+  So I can use the app
+
+  Scenario: User can see Appointment Confirmation when User create 2 Appointment
+    Given an user login with account "John Doe" and password as "ThisIsNotAPassword"
```

Comment:
an user login with account "John Doe" and password as "ThisIsNotAPassword" => `user login with account as "John Doe" and password as "ThisIsNotAPassword"`